ports/atmel-samd/boards/matrixportal_m4/pins.c

```diff
@@ -1,5 +1,41 @@
+#include "py/objtuple.h"
 #include "shared-bindings/board/__init__.h"
 
+STATIC const mp_rom_obj_tuple_t matrix_addr_tuple = {
+    {&mp_type_tuple},
+    5,
+    {
+        MP_ROM_PTR(&pin_PB07),
+        MP_ROM_PTR(&pin_PB08),
+        MP_ROM_PTR(&pin_PB09),
+        MP_ROM_PTR(&pin_PB15),
+        MP_ROM_PTR(&pin_PB13),
+    }
+};
+
+STATIC const mp_rom_obj_tuple_t matrix_data_tuple = {
+    {&mp_type_tuple},
+    6,
+    {
+        MP_ROM_PTR(&pin_PB00),
+        MP_ROM_PTR(&pin_PB01),
+        MP_ROM_PTR(&pin_PB02),
+
+        MP_ROM_PTR(&pin_PB03),
+        MP_ROM_PTR(&pin_PB04),
+        MP_ROM_PTR(&pin_PB05),
+    }
+};
+
+STATIC const mp_rom_map_elem_t matrix_common_table[] = {
+    { MP_OBJ_NEW_QSTR(MP_QSTR_rgb_pins),MP_ROM_PTR(&matrix_data_tuple) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_clock_pin),MP_ROM_PTR(&pin_PB06) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_latch_pin),MP_ROM_PTR(&pin_PB14) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_output_enable_pin),MP_ROM_PTR(&pin_PB12) },
+};
+MP_DEFINE_CONST_DICT(matrix_common_dict, matrix_common_table);
```

Comment:
This `MTX_COMMON` dictionary seems redundant, since the pins are already named as `board.SOMETHING`., and the clock, latch, and OE pins need to referenced separately anyway when passing them to the `RGBMatrix constructor`.

How about making `rgb_pins` be `MTX_RGB_PINS` at the top level of `board instead`, and removing `MTX_COMMON`?

Or is there some use for this dict I am not seeing?